browser/welcome/welcome.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 
-<html>
+<html data-theme="%UI_THEME%">
```

Comment:
this will not work as welcome.html isn't processed like  the main app, so variable will not be replaced

Reply:
change would be nedded in: `FileServerRequestHandler::preprocessWelcomeFile`